### PR TITLE
add ability to interpolate account_id and region in value filter

### PR DESCRIPTION
**Pull request by thisisshi**
Interpolate `{account_id` and `{region}` in ValueFilter

### CODE REVIEW

**Comment by alfredgamulo** [comment]
I think the format logic would be better placed in the `__init__`

File: c7n/filters/core.py [hunk 1]
```diff
@@ -281,6 +281,14 @@ class ValueFilter(Filter):
     annotate = True
 
     def __init__(self, data, manager=None):
+        try:
+            config_args = {
+                'account_id': manager.config.account_id,
+                'region': manager.config.region
+            }
+            self.data = format_string_values(data, **config_args)
+        except AttributeError:
```
Comment (kapilt) [inline c7n/filters/core.py]: account_id and region should always exist, so no need for for try/except

**Comment by kapilt** [comment]
I think this needs an issue outlining use cases, afaics all of them are covered without this functionality.

**Comment by kapilt** [comment]
closing for now, please open an issue for discussion on use cases if you want to pursue.

**Pull request closed without merging**